### PR TITLE
[WIP] PPCDEV-10155 User Group API documentation

### PR DESCRIPTION
**Pull request by mrfijas**

### CODE REVIEW

File: platform/index.rst [hunk 1]
```diff
@@ -8,3 +8,4 @@ Platform
   authorized_api/users/users_api
   authorized_api/apps/apps_api
   authorized_api/access_control/access_control_api
+  authorized_api/user_groups/user_groups_api
```
Comment (jzyinq) [inline platform/index.rst]: Can we rearrange that index alphabetically? 

obraz

It's small improvement but would look a lot better imo. Maybe with an exception for `Authorized Api Guide` as it's an introduction how to connect with our api. I would leave it at top.

File: platform/authorized_api/user_groups/v1.yaml [hunk 2]
```diff
@@ -0,0 +1,542 @@
...
+          type: string
+          default: name
+          required: false
+          enum:
+            - name
+            - created_at
+            - updated_at
+        - in: query
+          name: limit
+          description: Limits the number of returned user groups in response
```
Comment (jzyinq) [inline platform/authorized_api/user_groups/v1.yaml]: I remember we talked about generic descriptions for that kind of parameters e.g not `number returned user groups` but `number of entries`. Could we fix it here right or we leave it for another time? 

I think same thing could apply to `offset` as well.

File: platform/authorized_api/user_groups/v1.yaml [hunk 3]
```diff
@@ -0,0 +1,542 @@
...
+    in: query
+    name: search
+    description: Search phrase for searching for users by email or user id
+    type: string
+    required: false
+
+  UsersSort:
+    in: query
+    name: sort
+    description: Sort type that can be reversed by adding dash before type e.g (-email)
```
[the remark this answers is not in the document]
Comment (mrfijas) [reply platform/authorized_api/user_groups/v1.yaml]: yeah, definitely 👍
I've started fixing it, so I'm marking this PR as `[WIP]` and resurrect it later..
things like will be fixed soon: https://github.com/PiwikPRO/common-PPMS-AccessControl/blob/83cdac096bf148af1d13a4cd4b4b669ae9d6c334/docs/api/public/user-groups/v1.yaml#L65-L71

File: platform/authorized_api/user_groups/v1.yaml (hunk 2 above)
Comment (mrfijas) [reply platform/authorized_api/user_groups/v1.yaml]: yeah, it's fixed on my [PPCEV-10201](https://clearcodehq.atlassian.net/browse/PPCDEV-10201): https://github.com/PiwikPRO/common-PPMS-AccessControl/blob/83cdac096bf148af1d13a4cd4b4b669ae9d6c334/docs/api/public/user-groups/v1.yaml#L489-L503

**Comment by mrfijas** [comment]
ok, I'll make another PR after fixing docs in service, no need to [WIP] here..

**Pull request closed without merging**